README.md

```diff
@@ -1,70 +1,89 @@
 # UniPipe Service Broker
 
-A universal service broker to connect CI/CD pipelines with platforms that speak [OSB API](https://www.openservicebrokerapi.org/).
+A universal service broker to connect CI/CD pipelines with platforms that speak
+[OSB API](https://www.openservicebrokerapi.org/).
 
-This project implements the OSB API and stores information about instances and bindings in a git repository.
-The services will be created in a later step via a CI/CD pipeline that tracks the git repository.
-This decoupling allows using UniPipe Service Broker with any CI/CD piepline of your choice that will do the actual provisioning.
+This project implements the OSB API and stores information about instances and
+bindings in a git repository. The services will be created in a later step via a
+CI/CD pipeline that tracks the git repository. This decoupling allows using
+UniPipe Service Broker with any CI/CD pipeline of your choice that will do the
+actual provisioning.
 
 ![A marketplace that integrates with a Service Broker](assets/OSB-Graphic-03.png "Service Broker interactions")
 
 UniPipe Service Broker was formerly known as generic OSB API.
 
 ## Configuration
 
-The custom configuration can be done via environment variables. The following properties can be configured:
+The easiest way to deploy `Unipipe Service Broker` infrastructure and create
+your first `catalog.yml` and `handler.js` files can be done with our
```

Comment:
handler.js is specific to using `unipipe-cli transform` - this might be a bit too early to mention here

Reply:
i have removed the `handler.js` part